packages/firestore/test/unit/api/field_value.test.ts

```diff
@@ -33,4 +33,16 @@ describe('FieldValue', () => {
     expect(FieldValue.arrayUnion('a')).to.be.an.instanceOf(FieldValue);
     expect(FieldValue.arrayRemove('a')).to.be.an.instanceOf(FieldValue);
   });
+
+  it('supports JSON.stringify()', () => {
+    try {
+      JSON.stringify(FieldValue.delete());
+      JSON.stringify(FieldValue.serverTimestamp());
+      JSON.stringify(FieldValue.increment(1));
+      JSON.stringify(FieldValue.arrayUnion(2));
+      JSON.stringify(FieldValue.arrayRemove(3));
+    } catch (e) {
+      expect.fail('Unexpected error: ' + e.message);
```

Comment:
Can you just let the exception propagate to fail the test instead of catching it explicitly? As it is, the fail message will not indicate which call threw an exception but if the exception just propagates then I assume that a full stack trace would be provided, indicating which call failed.

Reply:
I wrote the test this way to ensure that it is obvious that I am only verifying that the API calls do not throw. Let me do this with a comment instead and remove the try/catch block.